boards/native/native_sim/Kconfig

```diff
@@ -18,12 +18,14 @@ comment "Native Simular (Single Core) options"
 
 config NATIVE_SIM_NATIVE_POSIX_COMPAT
 	bool "Pretend to be a native_posix board"
-	default y
+	default n
```

Comment:
The Kconfig guidelines state that we should not use `default n`. I'm fine with leaving it as is

Reply:
removed default :)